code-challenges.test.js

```diff
@@ -16,52 +16,232 @@
+    //  PASS  ./code-challenges.test.js
+    //   divisibleBy
+    //     ✓ should return a string indicating if the number is divisible by three or not (1 ms)
+
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // Pseudo code:
 
-// a) Create a test with expect statements for each of the variables provided.
+// input: takes in an array of words
+// output: returns an array with all the words capitolized
+// How? use .map() to iterate through the array of strings, and .toUpperCase to capitolize the strings.
 
-const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
-const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+// a) Create a test with expect statements for each of the variables provided.
+// describe("toCapitolize" , () => {
+//     it("takes in an array of strings" , () => {
+//         const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+
+//         const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+
+//         expect(toCapitolize(randomNouns1)).toEqual(["STREETLAMP", "POTATO" , "TEETH" , "CONCLUSION" , "NEPHEW"])
+//         expect(toCapitolize(randomNouns2)).toEqual(["TEMPERATURE" , "DATABASE" , "CHOPSTICKS" , "MANGO"]);
+//         })
+// })
+// const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+// // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+// const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
 
-// b) Create the function that makes the test pass.
+// FAIL  ./code-challenges.test.js
+// toCapitolize
+//   ✕ takes in an array of strings
+
+// ● toCapitolize › takes in an array of strings
+
+//   ReferenceError: toCapitolize is not defined
+  
+  // b) Create the function that makes the test pass.
+
+  describe("toCapitolize" , () => {
+    it("takes in an array of strings and returns and array with all values capitolized" , () => {
+        const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+
+        const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+
+        expect(toCapitolize(randomNouns1)).toEqual(["STREETLAMP", "POTATO" , "TEETH" , "CONCLUSION" , "NEPHEW"])
+        expect(toCapitolize(randomNouns2)).toEqual(["TEMPERATURE" , "DATABASE" , "CHOPSTICKS" , "MANGO"]);
+        })
+})
+
+const toCapitolize = (arr) => {
+    return arr.map(word => word.toUpperCase())
+    }
+//    Testing:
+//    console.log(toCapitolize(randomNouns1)) // output: [ 'STREETLAMP', 'POTATO', 'TEETH', 'CONCLUSION', 'NEPHEW' ]
+//    console.log(toCapitolize(randomNouns2)) // output: [ 'TEMPERATURE', 'DATABASE', 'CHOPSTICKS', 'MANGO' ]
```

Comment:
Pay attention to the expected output they give you in the prompt. It is not asking for the whole word to be capitalized, but only the first letter of each word!